app/controllers/vaccination_records/edit_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class VaccinationRecords::EditController < ApplicationController
+  before_action :set_vaccination_record
+  before_action :set_campaign
+
+  def edit_date_and_time
+    render :date_and_time
+  end
+
+  def update_date_and_time
```

Comment:
Couldn't these be `show` and `update`? I guess the idea is that we'll add more later?

Reply:
Yeah, that was the idea basically, I can rename them if you'd prefer?

Reply:
Hmm if we add more later it's fine